tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -195,23 +190,17 @@ export class WebGPUBackend extends KernelBackend {
 
   flushDisposalQueue() {
     this.tensorDisposalQueue.forEach(d => {
-      this.maybeReleaseBuffer(d);
+      this.releaseResource(d);
       this.tensorMap.delete(d);
     });
     this.uniformDisposalQueue.forEach(
-        d => this.bufferManager.releaseBuffer(d.buffer, d.byteSize, d.usage));
+        d => this.bufferManager.releaseBuffer(d.buffer, d.size, d.usage));
```

Comment:
LGTM.
Maybe we can merge releaseResource and releaseBuffer in the next change.

Reply:
releaseResource includes the buffer and texture management, so they don't need to be merged.